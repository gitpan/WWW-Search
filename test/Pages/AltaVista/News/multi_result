http://www.altavista.com/cgi-bin/query?what=news&text=yes&fmt=d&pg=q&q=%2BPerl+%2BCPAN
/cgi-bin/query?pg=q&what=news&fmt=d&text=yes&q=%2BPerl+%2BCPAN&stq=30&c9k
/cgi-bin/query?pg=q&what=news&fmt=d&text=yes&q=%2BPerl+%2BCPAN&stq=60&c9k
/cgi-bin/query?pg=q&what=news&fmt=d&text=yes&q=%2BPerl+%2BCPAN&stq=90&c9k
/cgi-bin/query?pg=q&what=news&fmt=d&text=yes&q=%2BPerl+%2BCPAN&stq=120&c9k
/cgi-bin/query?pg=q&what=news&fmt=d&text=yes&q=%2BPerl+%2BCPAN&stq=150&c9k
/cgi-bin/query?pg=q&what=news&fmt=d&text=yes&q=%2BPerl+%2BCPAN&stq=180&c9k
