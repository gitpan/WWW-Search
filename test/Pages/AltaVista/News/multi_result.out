http://ww2.altavista.com/cgi-bin/news?msg@187281@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@187532@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@168147@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@168181@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@168324@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@18238@comp%2elang%2eperl%2emodules
http://ww2.altavista.com/cgi-bin/news?msg@18240@comp%2elang%2eperl%2emodules
http://ww2.altavista.com/cgi-bin/news?msg@18271@comp%2elang%2eperl%2emodules
http://ww2.altavista.com/cgi-bin/news?msg@187670@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@188131@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@188249@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@188485@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@188631@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@189382@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@191957@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@191966@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@2055@comp%2elang%2eperl%2emoderated
http://ww2.altavista.com/cgi-bin/news?msg@8606@comp%2edatabases%2exbase%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@175355@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@182771@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@188007@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@20153@comp%2elang%2eperl%2emodules
http://ww2.altavista.com/cgi-bin/news?msg@163325@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@17168@comp%2elang%2eperl%2emodules
http://ww2.altavista.com/cgi-bin/news?msg@169073@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@169285@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@169351@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@1001@comp%2elang%2eperl%2emoderated
http://ww2.altavista.com/cgi-bin/news?msg@18261@comp%2elang%2eperl%2emodules
http://ww2.altavista.com/cgi-bin/news?msg@171377@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@171388@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@171423@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@171432@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@171956@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@18559@comp%2elang%2eperl%2emodules
http://ww2.altavista.com/cgi-bin/news?msg@18564@comp%2elang%2eperl%2emodules
http://ww2.altavista.com/cgi-bin/news?msg@18567@comp%2elang%2eperl%2emodules
http://ww2.altavista.com/cgi-bin/news?msg@174593@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@10025@comp%2elang%2eperl%2etk
http://ww2.altavista.com/cgi-bin/news?msg@177746@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@177984@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@178170@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@19318@comp%2elang%2eperl%2emodules
http://ww2.altavista.com/cgi-bin/news?msg@19728@comp%2elang%2eperl%2emodules
http://ww2.altavista.com/cgi-bin/news?msg@20156@comp%2elang%2eperl%2emodules
http://ww2.altavista.com/cgi-bin/news?msg@184830@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@186122@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@21059@comp%2elang%2eperl%2emodules
http://ww2.altavista.com/cgi-bin/news?msg@4151@fr%2ecomp%2elang%2eperl
http://ww2.altavista.com/cgi-bin/news?msg@5815@fr%2ecomp%2elang%2ec
http://ww2.altavista.com/cgi-bin/news?msg@172969@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@184859@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@10678@comp%2elang%2eperl%2etk
http://ww2.altavista.com/cgi-bin/news?msg@20704@comp%2elang%2eperl%2emodules
http://ww2.altavista.com/cgi-bin/news?msg@20732@comp%2elang%2eperl%2emodules
http://ww2.altavista.com/cgi-bin/news?msg@838@comp%2elang%2eperl%2eannounce
http://ww2.altavista.com/cgi-bin/news?msg@75822@comp%2eos%2evms
http://ww2.altavista.com/cgi-bin/news?msg@162105@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@56164@comp%2einfosystems%2ewww%2eauthoring%2ecgi
http://ww2.altavista.com/cgi-bin/news?msg@162387@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@162411@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@409@alt%2eperl
http://ww2.altavista.com/cgi-bin/news?msg@162584@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@410@alt%2eperl
http://ww2.altavista.com/cgi-bin/news?msg@162624@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@162642@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@162722@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@162777@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@17022@comp%2elang%2eperl%2emodules
http://ww2.altavista.com/cgi-bin/news?msg@162987@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@163017@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@46666@comp%2einfosystems%2ewww%2eservers%2eunix
http://ww2.altavista.com/cgi-bin/news?msg@163368@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@17110@comp%2elang%2eperl%2emodules
http://ww2.altavista.com/cgi-bin/news?msg@163459@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@163575@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@163585@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@198293@comp%2elang%2ejava%2eprogrammer
http://ww2.altavista.com/cgi-bin/news?msg@198438@comp%2elang%2ejava%2eprogrammer
http://ww2.altavista.com/cgi-bin/news?msg@164062@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@164085@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@429@alt%2eperl
http://ww2.altavista.com/cgi-bin/news?msg@17232@comp%2elang%2eperl%2emodules
http://ww2.altavista.com/cgi-bin/news?msg@164386@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@164388@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@164418@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@164439@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@164446@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@164493@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@164533@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@164596@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@164647@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@164674@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@164709@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@164806@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@164949@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@165125@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@17345@comp%2elang%2eperl%2emodules
http://ww2.altavista.com/cgi-bin/news?msg@165539@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@165568@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@17425@comp%2elang%2eperl%2emodules
http://ww2.altavista.com/cgi-bin/news?msg@165998@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@17458@comp%2elang%2eperl%2emodules
http://ww2.altavista.com/cgi-bin/news?msg@166089@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@166221@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@166287@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@166345@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@42925@comp%2edatabases%2ems%2dsqlserver
http://ww2.altavista.com/cgi-bin/news?msg@166479@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@166543@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@166578@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@166618@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@56689@comp%2einfosystems%2ewww%2eauthoring%2ecgi
http://ww2.altavista.com/cgi-bin/news?msg@808@comp%2elang%2eperl%2emoderated
http://ww2.altavista.com/cgi-bin/news?msg@167006@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@9193@comp%2elang%2eperl%2etk
http://ww2.altavista.com/cgi-bin/news?msg@9194@comp%2elang%2eperl%2etk
http://ww2.altavista.com/cgi-bin/news?msg@167114@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@824@comp%2elang%2eperl%2emoderated
http://ww2.altavista.com/cgi-bin/news?msg@167203@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@167205@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@56770@comp%2einfosystems%2ewww%2eauthoring%2ecgi
http://ww2.altavista.com/cgi-bin/news?msg@826@comp%2elang%2eperl%2emoderated
http://ww2.altavista.com/cgi-bin/news?msg@828@comp%2elang%2eperl%2emoderated
http://ww2.altavista.com/cgi-bin/news?msg@9223@comp%2elang%2eperl%2etk
http://ww2.altavista.com/cgi-bin/news?msg@9226@comp%2elang%2eperl%2etk
http://ww2.altavista.com/cgi-bin/news?msg@835@comp%2elang%2eperl%2emoderated
http://ww2.altavista.com/cgi-bin/news?msg@43106@comp%2edatabases%2ems%2dsqlserver
http://ww2.altavista.com/cgi-bin/news?msg@167390@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@167395@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@17656@comp%2elang%2eperl%2emodules
http://ww2.altavista.com/cgi-bin/news?msg@9233@comp%2elang%2eperl%2etk
http://ww2.altavista.com/cgi-bin/news?msg@167454@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@167498@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@43121@comp%2edatabases%2ems%2dsqlserver
http://ww2.altavista.com/cgi-bin/news?msg@167722@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@167961@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@168109@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@12762@comp%2etext%2esgml
http://ww2.altavista.com/cgi-bin/news?msg@168248@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@168260@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@168287@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@12783@comp%2etext%2esgml
http://ww2.altavista.com/cgi-bin/news?msg@9303@comp%2elang%2eperl%2etk
http://ww2.altavista.com/cgi-bin/news?msg@168439@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@12807@comp%2etext%2esgml
http://ww2.altavista.com/cgi-bin/news?msg@12809@comp%2etext%2esgml
http://ww2.altavista.com/cgi-bin/news?msg@168738@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@168748@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@9321@comp%2elang%2eperl%2etk
http://ww2.altavista.com/cgi-bin/news?msg@17883@comp%2elang%2eperl%2emodules
http://ww2.altavista.com/cgi-bin/news?msg@17948@comp%2eos%2eqnx
http://ww2.altavista.com/cgi-bin/news?msg@168829@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@168832@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@168888@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@56758@comp%2edatabases%2einformix
http://ww2.altavista.com/cgi-bin/news?msg@169065@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@169229@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@57105@comp%2einfosystems%2ewww%2eauthoring%2ecgi
http://ww2.altavista.com/cgi-bin/news?msg@9367@comp%2elang%2eperl%2etk
http://ww2.altavista.com/cgi-bin/news?msg@9368@comp%2elang%2eperl%2etk
http://ww2.altavista.com/cgi-bin/news?msg@169798@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@11337@comp%2egraphics%2eapps%2egnuplot
http://ww2.altavista.com/cgi-bin/news?msg@170076@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@18163@comp%2elang%2eperl%2emodules
http://ww2.altavista.com/cgi-bin/news?msg@171082@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@171203@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@171357@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@18377@comp%2elang%2eperl%2emodules
http://ww2.altavista.com/cgi-bin/news?msg@40081@comp%2elang%2epython
http://ww2.altavista.com/cgi-bin/news?msg@18458@comp%2elang%2eperl%2emodules
http://ww2.altavista.com/cgi-bin/news?msg@172005@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@172043@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@172356@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@7095@comp%2eos%2elinux%2epowerpc
http://ww2.altavista.com/cgi-bin/news?msg@173035@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@173242@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@173464@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@173655@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@173725@comp%2elang%2eperl%2emisc
http://ww2.altavista.com/cgi-bin/news?msg@173793@comp%2elang%2eperl%2emisc
